### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by esune**
Updated dependencies as far as possible without major rework to the project.

### CODE REVIEW

**Comment by swcurran** [comment]
6 digits changed on three lines.  Trivial :-).

**Comment by esune** [comment]
This is a first pass: any further changes will require upgrading to a newer version of node and, potentially, having to dig through additional code changes.

**Comment by esune** [comment]
API has been upgraded to use Node 18, working on frontends now. I will re-open the PR once I have things sorted out, with a list of hosted services that will/may need rebuilding.

**Comment by esune** [comment]
This took much longer than expected (especially the validation piece), but everything is up-to-date to the latest version that removes vulnerabilities and does not require major refactoring of the code.

⚠️ configurations for downstream projects will be updated soon, if new code in this repo triggers a build in the downstream repo it will likely fail until it is updated.

**Review by amanji — APPROVED** [review]

**Comment by esune** [comment]
Related PRs:
* https://github.com/bcgov/trust-over-ip-configurations/pull/125
* https://github.com/bcgov/essential-services-delivery/pull/128
* https://github.com/bcgov/identity-kit-configurations/pull/17

File: issuer-web/public/config/config.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "inviteRequired": false,
   "deepLinkPrefix":"didcomm://launch",
   "authentication": {
-    "enabled": true,
+    "enabled": false,
```
Comment (WadeBarnes) [inline issuer-web/public/config/config.json:13]: Intended change?

File: api/src/mongodb.ts [hunk 2]
```diff
@@ -8,15 +8,15 @@ function buildConnection(app: Application) {
...
+let db_status = false;
 
 export async function dbStatus(app: Application) {
   if (!db_status) { //if initial connection failed return 503
     return 503
   }
   const connection = buildConnection(app);
-  const client = new MongoClient(connection, { useUnifiedTopology: true, serverSelectionTimeoutMS: 1000 });
-  var db_code = 200;
+  const client = new MongoClient(connection, { serverSelectionTimeoutMS: 1000 });
```
Comment (WadeBarnes) [inline api/src/mongodb.ts:18]: Is `useUnifiedTopology: true` the default now?  This is part of a change made to fix some startup issues we were having with the issuer kit api.

**Review by WadeBarnes — COMMENTED** [review]
@esune, some questions below.

File: api/src/mongodb.ts (hunk 2 above)
Comment (esune) [reply api/src/mongodb.ts:18]: Yes, this is the default in the new(er) driver interface: https://www.mongodb.com/docs/drivers/node/current/whats-new/#mongoclientoptions-interface

File: issuer-web/public/config/config.json (hunk 1 above)
Comment (esune) [reply issuer-web/public/config/config.json:13]: No, that sneaked in. Not a concern though as it is the default file served with the app that is overwritten during deployment either way - used only for local development. Will revert to reduce noise.

**Review by WadeBarnes — APPROVED** [review]

**Pull request closed without merging**